core/primitives/scheduled_change.hpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_SCHEDULED_CHANGE_HPP
+#define KAGOME_CORE_PRIMITIVES_SCHEDULED_CHANGE_HPP
+
+#include <boost/optional.hpp>
+#include "primitives/common.hpp"
+#include "primitives/session_key.hpp"
+
+namespace kagome::primitives {
+  /// @struct ScheduledChange is returned by Grandpa::pending_change function
```

Comment:
Maybe it's worth to mention that GrandpaApi_grandpa_pending_change is a part of Runtime API, just to make it more clear.

Reply:
corrected comment